app/gui/src/model/project/synchronized.rs

```diff
@@ -810,7 +810,7 @@ mod test {
         let context_data = execution_context::plain::test::MockData::new();
         let Fixture { mut test, project, json_events_sender, .. } = Fixture::new(
             |mock_json_client| {
-                ExecutionFixture::mock_create_push_destroy_calls(&context_data, mock_json_client);
+                ExecutionFixture::mock_default_calls(&context_data, mock_json_client);
```

Comment:
This name change is drastic. Right now, I dont really understand what this funcion does. Would you mind explaining a little bit more what it does so we can find the suitable name for it?

Reply:
As we discussed on Discord, the documentation of the function contains information describing the specific calls mocked by it in more detail.